EIPS/eip-X.md

```diff
@@ -0,0 +1,220 @@
+  stateMutability: constant
+  outputs: 
+    - name: relayerChainId
+      type: uint256
+```
+
+### Events
+
+**Executed**
+
+MUST be emitted when a relayed message is executed.
+
+```solidity
+interface CrossChainReceiver {
+  event Executed(
+      uint256 indexed nonce,
+      Call[] calls
+  );
+}
+```
+
+```yaml
+- name: Executed
+  type: event
+  inputs:
+    - name: nonce
+      indexed: true
+      type: uint256
+    - name: calls
+      type: Call[]
+```
+
+**RelayerSet**
+
+MUST be emitted when the relayer is initialized or updated.
+
+```solidity
+interface CrossChainReceiver {
+  event RelayerSet(
+      address indexed relayer,
+      uint256 indexed chainId
+  );
+}
+```
+
+```yaml
+- name: RelayerSet
+  type: event
+  inputs:
+    - name: relayer
+      indexed: true
+      type: address
+    - name: relayerChainId
+      indexed: true
+      type: uint256
+```
+
+# Rationale
+
+There are some notable design decisions worth talking about:
+
+- Relayers and receivers have a one-to-many relationship
+- Calls are relayed in batches
+- Receivers include both events and accessors to access the relayer
+
+The message receiver always authenticates the sender. This is the case whether contracts live on the same chain or across chains. That's why the relayer is unaware of the receiver, but the receiver is aware of the relayer. This approach is the most compatible with transport layers (namely Nomad!) and affords the ability to broadcast messages to multiple receivers.
+
+Calls are relayed in batches because it is such a common action. Rather than have implementors take different approaches to encoding multiple calls into the `data` portion, this spec includes call batching to take away any guess work.
+
+The receiver makes the relayer address and chainId available on-chain, and emits events when the Relayer is initialized or set. This makes it easy to trace cross-chain execution. Thanks to the events and nonce, an indexer can know exactly who the relayer and receiver were in the past and trace the execution of a call batch. With the on-chain accessor, external contracts can check to see who the current relayer is.
+
+# Backwards Compatibility
+
+This specification is compatible with existing governance systems as it offers simple cross-chain execution.
+
+# Implementation
+
+A basic implementation of the interfaces is available on [Github](https://github.com/pooltogether/cross-chain-relay-implementation).
+
+# References
+
+This specification was born out of the conversation in [Ethereum Magicians](https://ethereum-magicians.org/t/eip-draft-multi-chain-governance/9284) and informed by prior work, including the Nomad [GovernanceRouter](https://github.com/nomad-xyz/monorepo/blob/14e99f292e20e08148b4b1b950d05c7fcb0f6cb9/packages/contracts-core/contracts/governance/GovernanceRouter.sol#L227) and the OpenZeppelin Labs [bridge](https://github.com/Amxx/openzeppelin-labs/tree/devel/crosschain-contracts/contracts/global/polygon-callbridge).
```

Comment:
```suggestion
```
Not a valid EIP section and external links should not be included in an EIP.  EIPs are for standardization; history, tutorials, best practices, etc. are not appropriate content for a standard and should live elsewhere.